.env.rob-test.json

```diff
@@ -1,5 +1,3 @@
 {
-	"PUBLIC_URL": "https://usaf.dps.mil/teams/10251/WAR-DEV/test/rob",
-	"REACT_APP_API_URL": "https://usaf.dps.mil/teams/10251/WAR-DEV",
 	"REACT_APP_TEST_SYS": true
 }
```

Comment:
Since this file is no longer used in any of the build scripts, you can safely delete it.